package.json

```diff
@@ -4,8 +4,8 @@
     "description": "",
     "main": "index.js",
     "scripts": {
-        "start": "node index.js",
-        "start-dev": "webpack -w --mode=development --env.NODE_ENV=development & nodemon index.js --env.NODE_ENV=development",
+        "start": "node --max-http-header-size 80000 index.js",
```

Comment:
Why did you pick 80,000?

Reply:
That was the max size previously.

Reply:
That was what the default was before the change - V8 of node this was set to 80kb, we should look to finding a lower magic number for this in the future.